controllers/construct/mongodbstatefulset.go

```diff
@@ -64,6 +64,17 @@ const (
 	automationAgentLogOptions = " -logFile ${AGENT_LOG_FILE} -maxLogFileDurationHrs ${AGENT_MAX_LOG_FILE_DURATION_HOURS} -logLevel ${AGENT_LOG_LEVEL}"
 
 	MongodbUserCommand = `current_uid=$(id -u)
+declare -r current_uid
```

Comment:
We could merge with below var, but this seems easy enough once we need to do one change in the future